src/native/corehost/host_runtime_contract.h

```diff
@@ -53,4 +59,17 @@ struct host_runtime_contract
         const char* entry_point_name);
 };
 
+#if defined(TARGET_ANDROID)
+struct host_configuration_property
+{
+    const char16_t* name;
+    const char16_t* value;
+};
+
+struct host_configuration_properties
+{
+    size_t nitems;
+    const host_configuration_property* data;
+};
+#endif
```

Comment:
```suggestion
```
Not needed anymore?

Reply:
Not now - I removed them. Note, anything defined here needs to be C friendly since it's shared w/ mono.